### PR TITLE
Paper subpage

### PR DESCRIPTION
**Pull request by richard-thonig**
As an exercise in csp.guru website manipulation I  added a page that lists papers and reports that reference csp.guru and changed the  config to display it.

### CODE REVIEW

**Comment by timtroendle** [comment]
This should be merged into the `website` branch rather than the `master` branch. I fixed it just now.

File: paper.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: Paper and Analysis
+---
+
+# Publications and analysis using CSP.guru
+
+csp. guru has beed developed since 2016 by Johan Lilliestam and his lab. The following publications use the data. If you want to cloaborate on additional publications or have used csp.guru for another publication not listed here, please [get in touch](mailto:richard.thonig@iass-potsdam.de).
```
Comment (timtroendle) [inline paper.md:8]: Typo on collaborate.

Comment (timtroendle) [inline paper.md:8]: Additional space between csp. and guru ;-)

File: paper.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: Paper and Analysis
+---
+
+# Publications and analysis using CSP.guru
```
Comment (timtroendle) [inline paper.md:6]: I guess this should be the title of the page. The title is given in line 3 however. So this line seems unnecessary to me.

File: paper.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+csp. guru has beed developed since 2016 by Johan Lilliestam and his lab. The following publications use the data. If you want to cloaborate on additional publications or have used csp.guru for another publication not listed here, please [get in touch](mailto:richard.thonig@iass-potsdam.de).
+
+## Peer-reviewed publications that produced a new version of CSP.guru and explain the data collection procedure.
```
Comment (timtroendle) [inline paper.md:10]: This is a very long headline that looks slightly odd in the rendered website. I'd shorten it. Maybe to "Peer-reviewed publications explaining data collection"

**Review by timtroendle — APPROVED** [review]
Looks good. Just a few small comments.

**Comment by timtroendle** [comment]
Fixes #61

**Comment by richard-thonig** [comment]
> 
> 
> This should be merged into the `website` branch rather than the `master` branch. I fixed it just now.
Komisch, ich dachte ich war auf dem website branch, aber er wollte dann changes vom master übernehmen - ich dachte das wäre nötig und hab ja geklickt.

File: paper.md (hunk 3 above)
Comment (richard-thonig) [reply paper.md]: My problem is, is that I don't "see what I get" locally.

**Pull request merged**